platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -562,6 +611,16 @@ optional<std::pair<Response, uint64_t>> OfflineDatabase::getRegionResource(int64
     return response;
 }
 
+optional<int64_t> OfflineDatabase::hasRegionResource(int64_t regionID, const Resource& resource) {
```

Comment:
Please add a test for this [here](https://github.com/mapbox/mapbox-gl-native/blob/master/test/storage/offline_database.test.cpp).